compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.5"
+
+networks:
+  sliding-sync:
+
+services:
+  slidingsync:
+    image: ghcr.io/matrix-org/sliding-sync
```

Comment:
Probably should have a versioned tag.

Reply:
this means that it has to be incremented regularly, but it makes sense obviously to have it... though I was unsure to how to handle this before a stable version of the project. As soon as there are major versions I guess we can simply pin it to a major version...
should I add an exact version tag for now ?